.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+---
+repos:
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.35.1
+    hooks:
+      - id: yamllint
```

Comment:
Let's feed config to our yamllint with below content:

```sh
extends: default

yaml-files:
  - '*.yaml'
  - '*.yml'
  - '.yamllint'
  -
# see rules
# https://yamllint.readthedocs.io/en/stable/rules.html
rules:
  line-length: disable
  document-start: disable
  comments: disable
```

Reply:
Done